plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1728,6 +1729,12 @@ public void setEnvVariables(String[] env) {
 
         initialEnv.put("__PYDEV_CONDA_PREFIX__", condaPrefix.toString());
         initialEnv.put("__PYDEV_CONDA_DEFAULT_ENV__", condaPrefix.getName());
+
+        File condaExec = PyDevCondaPreferences.findCondaExecutable(this);
+        File condaScriptsDir = condaExec.getParentFile();
+        File condaActivation = new File(condaScriptsDir, "activate.bat");
```

Comment:
We should also check linux here... if `condaActivation` is not found, we should at least log it.